runtime/pom.xml

```diff
@@ -65,15 +40,19 @@
                             There should be no newlines in the classes list, otherwise the filter silently
                             fails at runtime
                             -->
-                            <resources>dev/hilla/ConditionalOnFeatureFlag.class,dev/hilla/EndpointControllerConfiguration.class,dev/hilla/FeatureFlagCondition.class,dev/hilla/startup/EndpointsValidator.class,dev/hilla/push/PushConfigurer.class</resources>
+                            <resources>
+                                dev/hilla/ConditionalOnFeatureFlag.class,dev/hilla/EndpointControllerConfiguration.class,dev/hilla/FeatureFlagCondition.class,dev/hilla/startup/EndpointsValidator.class,dev/hilla/push/PushConfigurer.class
+                            </resources>
```

Comment:
Still working?

Reply:
Fixed

Reply:
Resources config in specified only on `runtime` and `runtime-react` modules.
They are both fixed now